shared/tests/utility/math/filter/TestFilters.cpp

```diff
@@ -245,12 +245,12 @@ TEST_CASE("Test the ParticleFilter", "[utility][math][filter][ParticleFilter]")
 
     INFO("Configuring the ParticleFilter with")
     INFO("    Time step..........: " << deltaT);
-    INFO("    Number of Particles: " << number_of_particles)
+    INFO("    Number of Particles: " << model_filter.model.getParticleCount());
```

Comment:
Need to set this in the model
https://github.com/NUbots/NUbots/blob/f03da89d6b52344603e8260032eb58284ea6e4dd/shared/tests/utility/math/filter/TestFilters.cpp#L168